<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AlgorithmRoomGUIClass</class>
 <widget class="QMainWindow" name="AlgorithmRoomGUIClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1033</width>
    <height>668</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AlgorithmRoomGUI</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTreeWidget" name="treeWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>131</width>
      <height>621</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>类型</string>
     </property>
    </column>
    <item>
     <property name="text">
      <string>SORT</string>
     </property>
     <property name="flags">
      <set>ItemIsSelectable|ItemIsUserCheckable|ItemIsEnabled</set>
     </property>
    </item>
    <item>
     <property name="text">
      <string>SEARCH</string>
     </property>
    </item>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>0</y>
      <width>901</width>
      <height>621</height>
     </rect>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QListWidget" name="listWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>141</width>
        <height>621</height>
       </rect>
      </property>
     </widget>
     <widget class="QGraphicsView" name="graphicsView">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>0</y>
        <width>761</width>
        <height>621</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QListWidget" name="listWidget_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>141</width>
        <height>621</height>
       </rect>
      </property>
     </widget>
     <widget class="QGraphicsView" name="graphicsView_2">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>0</y>
        <width>761</width>
        <height>621</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1033</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="algorithmroomgui.qrc"/>
 </resources>
 <connections/>
</ui>
